test/object-store/sync/app.cpp

```diff
@@ -2219,7 +2219,7 @@ TEST_CASE("app: sync integration", "[sync][app]") {
                     logger->trace("request.url (%1): %2", request_count, request.url);
                     REQUIRE(request.url.find("somehost:9090") != std::string::npos);
                     redir_transport->simulated_response = {
-                        301, 0, {{"Location", redirect_url}, {"Content-Type", "application/json"}}, "Some body data"};
+                        308, 0, {{"Location", redirect_url}, {"Content-Type", "application/json"}}, "Some body data"};
```

Comment:
nit: shouldn't we have the test for 301 as well? (i.e, test both values)

Reply:
The redirect test covers a mixture of 301 and 308 responses